app/controllers/well_known_controller.rb

```diff
@@ -0,0 +1,27 @@
+class WellKnownController < ApplicationController
+  before_action :set_default_response_format
+
+  def keybase
+    return render json: {} unless Keybase.enabled?
+    @domain = Keybase.DOMAIN
+    @name = Rails.application.name
+    @brand_color = "#AC130D"
+    @description = "Computing-focused community centered around link aggregation and discussion"
+    @contacts = ["admin@#{Keybase.DOMAIN}"]
+    # rubocop:disable Style/FormatStringToken
```

Comment:
Huh, why did this cop fire? I've read [its docs](https://rubocop.readthedocs.io/en/stable/cops_style/#styleformatstringtoken) and I don't even get why it's firing in this case. I see you do `%{}` instead of `#{}` for some values, but I don't see a call to `format` or `%` in this code or view, so I'm puzzled why you do that?

Reply:
sure. `%{}` is what keybase is expecting, so we need to do that. and here's the error rubocop gives...
```
C: Style/FormatStringToken: Prefer annotated tokens (like %<foo>s) over template tokens (like %{foo}).
```